### DISCUSSION TITLE
[Feature] Please remove the username/password login, when this is disabled in web, on mobile.

### DISCUSSION THREAD

**Discussion by Dental1347** [immich-app/immich#2206] [Feature Request]
### The feature

Good evening,

I would like to request a feature to remove the "normal" login, being the username and password when this has been disabled in the web app. (Administration > Setttings > Password Authentication > disable login with email and password)

In the web interface it looks like this:
![image](https://user-images.githubusercontent.com/44779473/230658018-6057fa43-7782-44e2-b2d9-42df41fbae89.png)

While the interface on mobile looks like this: 
![image](https://user-images.githubusercontent.com/44779473/230658054-c99236ca-f0a7-48ca-96db-5014cb92aa1a.png)

NOTE: It is not possible to login with username/password, as this has been disabled and generates an error.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by jrasm91** [comment]
This was implemented a few releases ago
